requirements/dev.txt

```diff
@@ -46,22 +49,24 @@ pylint==2.6.0             # via -r requirements/quality.txt, edx-lint, pylint-ce
 pyparsing==2.4.7          # via -r requirements/ci.txt, -r requirements/test.txt, packaging
 pytest-cov==2.10.1        # via -r requirements/test.txt
 pytest-django==4.1.0      # via -r requirements/test.txt
-pytest==6.2.1             # via -r requirements/test.txt, pytest-cov, pytest-django
-pytz==2020.4              # via -r requirements/test.txt, celery, django
+pytest==6.1.2             # via -r requirements/test.txt, pytest-cov, pytest-django
```

Comment:
Seems like some dependencies have downgraded

Reply:
Last PR https://github.com/edx/django-user-tasks/pull/104 merged by Awais bhae had the `make upgrade` ran on `python38` but this job ran on `python35` which caused the downgrading of package versions.